CONTRIBUTING.md

```diff
@@ -6,23 +6,23 @@ welcome to contribute towards development in the form of peer review, documentat
 testing and patches.
 
 Anyone is invited to contribute without regard to technical experience, "expertise", OSS
-experience, age, or other concern. Though developing cryptocurrencies demand a
+experience, age, or other concern. However, developing cryptocurrencies demand a
```

Comment:
grammar ultra nit: I think `demand` should be `demands` as the subject is the gerund `developing`.

This could also be rewritten as: `However, the development of cryptocurrencies demands...`

Reply:
thank you and fixed!